common/djangoapps/config_models/models.py

```diff
@@ -136,9 +145,9 @@ def current(cls, *args):
         return current
 
     @classmethod
-    def is_enabled(cls):
+    def is_enabled(cls, *args):
```

Comment:
@bdero `is_enabled` is almost always called with single parameter `site` - do you think it would make sense to promote it to an actual parameter? Also, it might be good to give it a default value of current site. SOmething like

    def is_enabled(site=None, * args):
       site = site if site else get_current_site()
       return cls.current(site, * args),enabled